backend/src/main/java/com/capstone/moneytree/controller/UserController.java

```diff
@@ -197,4 +197,9 @@ public User login(@RequestBody User credentials) throws CredentialNotFoundExcept
    public ResponseEntity<List<User>> getTopUserForStock(@PathVariable String symbol) {
       return ResponseEntity.ok(userService.getTopUsers(symbol));
    }
+
+   @GetMapping("{id}/owned_by_followers/{symbol}")
+   public ResponseEntity<List<User>> getStockOwnedByFollowers(@PathVariable Long id, @PathVariable String symbol) {
+      return ResponseEntity.ok(userService.getFollowersWhoOwnsTheStock(id, symbol));
+   }
```

Comment:
We could settle on the naming of that endpoint not satisfied with what I chose

Reply:
naming is fine, it clearly conveys what this endpoints returns even if it is not sexy.